src/styles/base.css

```diff
@@ -135,3 +135,7 @@ dialog {
 p {
   text-wrap: pretty;
 }
+
+button:where(.astro-jif73yzw) {
```

Comment:
Hm, I think the classname `.astro-<id>` is autogenerated and won't be consistent, no?

Reply:
Or is it consistent? I see it works on the preview deployment